pytest.ini

```diff
@@ -7,7 +7,6 @@ addopts =
 	# Mark some tests not to be run
 	-m "not functional"
 markers =
-	dbus: subscription-manager tests for DBus.
```

Comment:
I think that this: `py.test -m "dbus"` was useful, when you wanted to test only dbus code (or `py.test -m "not dbus"`, when you did not). I would keep this.

Reply:
I don't use `py.test -m dbus`. So feel free to remove it.